config/packages/messenger.yaml

```diff
@@ -1,24 +1,15 @@
 framework:
     messenger:
-        failure_transport: failed
-
         transports:
-            # https://symfony.com/doc/current/messenger.html#transport-configuration
-            async:
+            async_mail:
+                dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+                options:
+                    queue_name: mail
+            async_sms:
                 dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
                 options:
-                    use_notify: true
-                    check_delayed_interval: 60000
-                retry_strategy:
-                    max_retries: 3
-                    multiplier: 2
-            failed: 'doctrine://default?queue_name=failed'
-            sync: 'sync://'
+                    queue_name: sms
 
         routing:
-            Symfony\Component\Mailer\Messenger\SendEmailMessage: sync
-            Symfony\Component\Notifier\Message\ChatMessage: sync
-            Symfony\Component\Notifier\Message\SmsMessage: sync
-
-            # Route your messages to the transports
-            # 'App\Message\YourMessage': async
+            Symfony\Component\Mailer\Messenger\SendEmailMessage: async_mail
```

Comment:
I assume you have the infinite loop issue. Did you test this? Are emails sent?

Reply:
I have tested it again now.